catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/SubjectCommands.java

```diff
@@ -68,46 +64,59 @@
    * the user doesn't have the permission to run the command.
    *
    * @return value returned by {@link #executeWithSubject()}, or {@code null} if the command failed
-   * @throws InvocationTargetException thrown if an exception occurred while executing the command
-   * @throws Exception if any other unexpected exception occurred
    */
   @Override
-  public Object execute() throws Exception {
-
-    try {
-      if (isNotBlank(user)) {
-        return runWithUserName();
-      } else {
-        return security.runWithSubjectOrElevate(
-            () ->
-                AccessController.doPrivileged(
-                    (PrivilegedExceptionAction<Object>) this::executeWithSubject));
-      }
-    } catch (SecurityServiceException e) {
-      printErrorMessage(e.getMessage());
-    } catch (InvocationTargetException e) {
-      printErrorMessage(e.getCause().getMessage());
+  public Object execute() {
+    if (isNotBlank(user)) {
+      return runWithUsername();
+    } else {
+      return runWithoutUsername();
     }
-    return null;
   }
 
-  private Object runWithUserName() throws InvocationTargetException {
+  @Nullable
+  private Object runWithUsername() {
+    final String password;
     try {
-      String password = session.readLine("Password for " + user + ": ", '*');
-      Subject subject = security.getSubject(user, password, "127.0.0.1");
+      password = session.readLine("Password for " + user + ": ", '*');
+    } catch (IOException e) {
+      LOGGER.info("Failed to run command", e);
+      printErrorMessage("Failed to read password");
+      return null;
+    }
 
-      if (subject == null) {
-        printErrorMessage("Invalid username/password");
-        return null;
-      }
+    final Subject subject = security.getSubject(user, password, "127.0.0.1");
+    if (subject == null) {
+      printErrorMessage("Invalid username/password");
+      return null;
+    }
 
+    try {
       return subject.execute(this::executeWithSubject);
     } catch (ExecutionException e) {
-      LOGGER.info("Failed to run command: {}", e.getCause().getMessage(), e.getCause());
-      throw new InvocationTargetException(e.getCause());
-    } catch (IOException e) {
-      LOGGER.info("Failed to run command", e);
-      printErrorMessage("Failed to read password");
+      final Throwable failure = e.getCause();
+      final String message = failure.getMessage();
+      LOGGER.info("Failed to run command: {}", message, failure);
```

Comment:
❗ missing placeholder for logging parameter

Reply:
Looks like is prints it correctly as is. Is there something I'm not seeing?
```
admin@root()> catalog:range --user admin name ___ ___
Password for admin: *****

#       ID                                name                       Title

Could not parse date parameters.
admin@root()> log:tail
12:30:20,670 | INFO  | min name ___ ___ | ice.ddf.commands.catalog.SubjectCommands   99 | atalog-core-commands | Failed to run command: Could not parse date parameters.
java.lang.InterruptedException: Could not parse date parameters.
	at org.codice.ddf.commands.catalog.RangeCommand.executeWithSubject(RangeCommand.java:127) ~[bundleFile:?]
	at org.apache.shiro.subject.support.SubjectCallable.doCall(SubjectCallable.java:90) ~[?:?]
	at org.apache.shiro.subject.support.SubjectCallable.call(SubjectCallable.java:83) ~[?:?]
	at org.apache.shiro.subject.support.DelegatingSubject.execute(DelegatingSubject.java:387) ~[?:?]
	at org.codice.ddf.commands.catalog.SubjectCommands.runWithUsername(SubjectCommands.java:95) [bundleFile:?]
	at org.codice.ddf.commands.catalog.SubjectCommands.execute(SubjectCommands.java:71) [bundleFile:?]
	...
```

Reply:
my bad - you are logging the exception as well as the message - disregard!